api/users/views.py

```diff
@@ -103,7 +103,6 @@ class UserList(JSONAPIBaseView, generics.ListCreateAPIView, ODMFilterMixin):
     """
     permission_classes = (
         drf_permissions.IsAuthenticatedOrReadOnly,
-        base_permissions.RequiresScopedRequestOrReadOnly,
```

Comment:
I don't think removing this is correct; I think we just want to remove the cookie requirement for this permission class. We do want it to be either scoped, readonly, or cookie.